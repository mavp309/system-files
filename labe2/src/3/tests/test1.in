6 2 42 3 42 4 42 1192 2147483646 1193 2147483646 1194 2147483646
