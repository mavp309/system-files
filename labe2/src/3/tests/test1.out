511104 359209 298816 849344 147249 622656 
